From 7b2424443c763c5d4578dc2743f6493513be00e5 Mon Sep 17 00:00:00 2001
From: Weston Harper <13283469+wesharper@users.noreply.github.com>
Date: Thu, 13 Feb 2025 20:34:26 -0700
Subject: [PATCH] add support for additional Razer Wolverine controllers

---
 xpad.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/xpad.c b/xpad.c
index afacca8..a937edc 100644
--- a/xpad.c
+++ b/xpad.c
@@ -80,7 +80,7 @@

 #define XPAD_PKT_LEN 64

-/* The Guitar Hero Live (GHL) Xbox One dongles require a poke
+/* The Guitar Hero Live (GHL) Xbox One dongles require a poke
  * every 8 seconds.
  */
 #define GHL_GUITAR_POKE_INTERVAL 8 /* In seconds */
@@ -325,6 +325,8 @@ static const struct xpad_device {
 	{ 0x1532, 0x0a00, "Razer Atrox Arcade Stick", MAP_TRIGGERS_TO_BUTTONS, XTYPE_XBOXONE },
 	{ 0x1532, 0x0a03, "Razer Wildcat", 0, XTYPE_XBOXONE },
 	{ 0x1532, 0x0a29, "Razer Wolverine V2", 0, XTYPE_XBOXONE },
+	{ 0x1532, 0x0a43, "Razer Wolverine V3 Tournament Edition", MAP_PADDLES, XTYPE_XBOXONE },
+	{ 0x1532, 0x0a3f, "Razer Wolverine V3 Pro 2.4", MAP_PADDLES, XTYPE_XBOX360 },
 	{ 0x15e4, 0x3f00, "Power A Mini Pro Elite", 0, XTYPE_XBOX360 },
 	{ 0x15e4, 0x3f0a, "Xbox Airflo wired controller", 0, XTYPE_XBOX360 },
 	{ 0x15e4, 0x3f10, "Batarang Xbox 360 controller", 0, XTYPE_XBOX360 },
--
2.47.2

